build-tools/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM amd64/ubuntu:bionic as builder
+WORKDIR /usr/src/app
+RUN apt-get -y update
+RUN apt-get -y install build-essential
+RUN apt-get -y install openssl pkg-config libssl-dev
+RUN apt-get -y install gcc
+RUN apt-get -y install curl
+
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+RUN . $HOME/.cargo/env
+COPY Cargo.toml .
+COPY Cargo.lock .
+RUN mkdir ./src && echo 'fn main() { println!("Dummy!"); }' > ./src/main.rs
+RUN /root/.cargo/bin/cargo build --release
+RUN rm -rf ./src
+COPY src ./src
+RUN touch -a -m ./src/main.rs
+RUN /root/.cargo/bin/cargo build --release
+
+FROM amd64/ubuntu:bionic as stripper
+RUN apt-get update -y
+RUN apt-get install -y binutils
+COPY --from=builder /usr/src/app/target/release/build_tools /bin
+RUN strip /bin/build_tools
+
+# strip the binary and zip it
+FROM amd64/ubuntu:bionic as compresser
+RUN apt-get update
+RUN apt-get install -y zip
+RUN mkdir -p /bin
+WORKDIR /bin
+COPY --from=stripper /bin/build_tools /bin/build_tools
```

Comment:
Why do strip our build tools? Wouldn't debug information be helpful?

Reply:
you're right, it's not stripped anymore